### ISSUE TITLE
Support PR title and body

### ISSUE THREAD

**Issue by cbaxter713** [paygoc6/action-pull-request-another-repo#21]
It would be great to have options supported to populate the generated PR title and body copy

**Comment by bishabosha** [comment]
I agree - in particular it is very useful to have a record on the generated PR why it was made.
